internal/rpc/jsonrpc/methods.go

```diff
@@ -4291,10 +4296,12 @@ func (s *Server) ticketInfo(ctx context.Context, icmd interface{}) (interface{},
 			if err != nil {
 				return false, err
 			}
-			info.Choices = make([]types.VoteChoice, len(choices))
-			for i := range choices {
-				info.Choices[i].AgendaID = choices[i].AgendaID
-				info.Choices[i].ChoiceID = choices[i].ChoiceID
+			info.Choices = make([]types.VoteChoice, 0)
```

Comment:
same here, and making a slice with zero length zero cap isn't a good use of make()